ruby-sinatra/README.md

```diff
@@ -13,6 +13,6 @@ sudo apt update && sudo apt install libpq-dev
 ### Setup
 
 - Install required gems with `bundle install`
-- Initial database with `bundle exec rake db:create`
-- Run migrations `bundle exec rake db:migrate`
+- Initial database with `bundle exec rake db:create` - If the DB doesn't Exists!!!
```

Comment:
I am not sure I understand this. When should this command be run? When the db-container is running or?

Reply:
If our db container is running it exists heheh So you dont need to run it.

Reply:
I will delete it.